src/sub-blocks/Content/Content.tsx

```diff
@@ -67,10 +68,11 @@ const Content = (props: ContentProps) => {
                     title={titleProps}
                     colSizes={{all: 12}}
                     id={titleId}
+                    {...a11yProps?.title}
```

Comment:
why we need `a11yProps` if we have static role for element ?title always title, list always list and e.t.c

Reply:
In this specific case its here to give `Title` `role="link"`